server/authorization.go

```diff
@@ -42,7 +42,7 @@ func (p *Plugin) authenticateAndFetchUser(userID, userEmail, channelID string) (
 		return nil, &authError{Message: oauthMsg, Err: apiErr}
```

Comment:
Do we want to watch out for the same issue here?

Reply:
With the current usage, Err will be nil, but that will not be any problem, since authError will be a non-nil value.

Reply:
So if `apiErr` is nil, then we don't have anything to log right? Just so I understand, we do need to return and log an error (`authError.Err`) when we fail to fetch the graph user, but not when the user isn't found in the store.